src/main/ts/commands/Revive.ts

```diff
@@ -0,0 +1,70 @@
+import * as O from 'fp-ts/Option';
+import { pipe } from 'fp-ts/function';
+import { SimpleGit } from 'simple-git';
+import { ReviveArgs } from '../args/BeehiveArgs';
+import { createReleaseBranch, getBranchDetails, getReleaseBranchName, versionFromReleaseBranch } from '../core/BranchLogic';
+import * as AdvanceVersion from '../core/AdvanceVersion';
+import { printHeaderMessage } from '../core/Messages';
+import * as Version from '../core/Version';
+import * as ArrayUtils from '../utils/ArrayUtils';
+import { Comparison } from '../utils/Comparison';
+import * as OptionUtils from '../utils/OptionUtils';
+import * as PromiseUtils from '../utils/PromiseUtils';
+import * as Git from '../utils/Git';
+
+type MajorMinorVersion = Version.MajorMinorVersion;
+type Version = Version.Version;
+
+interface TagDetails {
+  readonly version: Version;
+  readonly name: string;
+}
+
+const getLatestTag = async (git: SimpleGit, version: MajorMinorVersion): Promise<TagDetails> => {
```

Comment:
Maybe we could call this `getLatestTagForVersion` or similar? I realise it can be inferred given it takes in `version` but it confused me a little

Reply:
Fixed